client/web/src/repo/blob/use-blob-store.ts

```diff
@@ -0,0 +1,30 @@
+import { EditorView } from '@codemirror/view'
+import create from 'zustand'
+
+import { lineScrollEnforcing, SelectedLineRange, setSelectedLines } from './codemirror/linenumbers'
+
+interface BlobStore {
+    editView: EditorView | null
+}
+
+export const useBlobUIStore = create<BlobStore>(() => ({ editView: null }))
```

Comment:
To make sure that it's not used directly if that's the correct intention.

```suggestion
const useBlobUIStore = create<BlobStore>(() => ({ editView: null }))
```